src/uhttp.c

```diff
@@ -320,7 +325,48 @@ static int write_text_line(HTTP_CLIENT_HANDLE_DATA* http_data, const char* text_
         result = 0;
         if (http_data->trace_on)
         {
-            LOG(AZ_LOG_TRACE, LOG_LINE, "%s", text_line);
+            char* authStart = strstr(text_line, AUTH_HEADER);
+            char* proxyAuthStart = strstr(text_line, PROXY_AUTH_HEADER);
+            char* authEol = strstr(authStart, HEADER_ENDING);
```

Comment:
What happen if AUTH_HEADER or PROXY_AUTH_HEADER is not in the header? you are passing null to strstr()

The ISO C standard says that the behavior is undefined.

Reply:
Fixed.